examples/offline_inference.py

```diff
@@ -1,20 +1,35 @@
+import time
+
 from vllm import LLM, SamplingParams
 
+template = ("Below is an instruction that describes a task. Write a response "
+            "that appropriately completes the request.\n\n### Instruction:\n{}"
+            "\n\n### Response:")
+
 # Sample prompts.
 prompts = [
     "Hello, my name is",
     "The president of the United States is",
     "The capital of France is",
     "The future of AI is",
 ]
+prompts = [template.format(prompt) for prompt in prompts]
 # Create a sampling params object.
 sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
 
 # Create an LLM.
-llm = LLM(model="facebook/opt-125m")
+llm = LLM(model="ibm-granite/granite-7b-instruct",
+          use_v2_block_manager=True,
+          enforce_eager=True,
```

Comment:
Does MLPSpeculator work with Cudagraph now?

Reply:
@LiuXiaoxuanPKU I'm actually not sure ... it's still pretty fast without it. Was thinking to look at cudagraph as a follow-on.